src/util/sss_ldap.c

```diff
@@ -55,6 +55,22 @@ int sss_ldap_get_diagnostic_msg(TALLOC_CTX *mem_ctx, LDAP *ld, char **_errmsg)
     return EOK;
 }
 
+void sss_ldap_error_debug(int level, LDAP *ld, int error_code)
+{
+    char *diagnostics = NULL;
+    int ret;
+
+    ret = sss_ldap_get_diagnostic_msg(ld, ld, &diagnostics);
```

Comment:
There is an issue with this patch:
```
[_sss_talloc_log_fn] (0x0010): Bad talloc magic value - unknown value
```
Likely `LDAP *ld` isn't proper talloc context.